### DISCUSSION TITLE
Hide suggestions or mark as not interested in for you tab

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [community/community#13277] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm not interested in following accounts in shows under "Recommended for you", so I would like a way to hide suggestions or mark as not interested, like YT has so it doesn't show it to me again:

![image](https://user-images.githubusercontent.com/84712013/159543191-99ed3400-b0f1-4dae-a1e8-0ed702d76e79.png)

**Comment by RundesBalli** [comment]
This partly agrees with my suggestion, which is why I would like to link it here:
https://github.com/github/feedback/discussions/13143

**Reply by KendallDoesCoding** [reply]
Thanks, I made a suggestion a while ago to do this in the explore tab but now making one for only the "For You" tab but it can be implemented in all instances.

**Comment by sntwandy** [comment]
It would be interesting to make a global ranking of the content that we like the most and that we like the least so we know what to show, or do it for each user so that we have a personalized feed but without losing globality.

**Comment by jfullstackdev** [comment, marked as answer]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by KendallDoesCoding** [reply]
Hey @jdevstatic ,

Amazing. Thanks for sharing.

**Comment by NitinIsFit** [comment]
> hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters
> 
> ![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

How soon do you think we will receive this update on mobile.

**Comment by KendallDoesCoding** [comment]
In a few weeks?
